### ISSUE TITLE
font-weight-notation also should ignore sass (scss) valiable

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#746]
```scss
.btn {
    font-weight: $btn-font-weight;
}
```

**Comment by davidtheclark** [comment]
Thanks for both reports, @evilebottnawi.
